README.md

```diff
@@ -534,22 +550,26 @@ spec:
   versions:
   - version: "*"
     containers:
-    - .spec.template.spec.containers
-    - .spec.template.spec.initContainers
-    volumes: .spec.template.spec.volumes
+    - path: .spec.template.spec.containers[*]
+      name: .name
+      env: .env
+      volumeMounts: .volumeMounts
+    - path: .spec.template.spec.initContainers[*]
+      name: .name
+      env: .env
+      volumeMounts: .volumeMounts
+    volumes: .spec/template.spec.volumes
```

Comment:
@scothis - did we miss a `/` replacement here?

Reply:
```suggestion
    volumes: .spec.template.spec.volumes
```